x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_table/intance_details.tsx

```diff
@@ -42,8 +48,20 @@ interface Props {
   kuery: string;
 }
 
-function toKeyValuePairs(keys: string[], data: ServiceInstanceDetails) {
-  return keys.map((key) => ({ key, value: get(data, key) }));
+function toKeyValuePairs({
+  keys,
+  data,
+  showFilterByOption = true,
```

Comment:
nit: I didn't immediately understand the meaning of `showFilterByOption`. Perhaps we can reword it to `isFilterable`:

```suggestion
  isFilterable = true,
```

or simply

```suggestion
  filterable = true,
```

Reply:
Updated here: de9d270